### PR TITLE
Elaborated timeout error message

### PR DESCRIPTION
**Pull request by alfmags**

### CODE REVIEW

File: vipps-psp-api.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ Once user has confirmed number the payment can be considered initiated.
 
 ### Skip landing page
 
-*Only available for whitelisted sale units.*
+_Only available for whitelisted sale units._
```
Comment (cloveras) [inline vipps-psp-api.md:133]:
```suggestion
**Please note:** This feature has to be specially enabled by Vipps for eligible sale units: The sale units must be whitelisted by Vipps. Skipping the landing page is typically used at physical points of sale where there is no display available.
```

**Review by cloveras — CHANGES_REQUESTED** [review]
Looks good! Comments: Nothing substantial.

File: vipps-psp-api.md [hunk 2]
```diff
@@ -117,6 +117,7 @@ in the Getting started guide, for details.
 When a user chooses Vipps as the payment method, a payment request is initiated
 by the PSP to the Vipps API. Vipps creates the payment and returns a link that does the following:
 
+
```
Comment (rebekaburnett) [inline vipps-psp-api.md:120]:
```suggestion
```

File: vipps-psp-api.md [hunk 3]
```diff
@@ -493,6 +496,7 @@ Initializing a recurring payment works in the same way as a non-recurring paymen
 but with the inclusion of a `scope` and `agreementURL` in the init call.
 
 At that time, the `scope` can be set to `psp_subscription`.
+
```
Comment (rebekaburnett) [inline vipps-psp-api.md:499]:
```suggestion
```

**Review by rebekaburnett — APPROVED** [review]
I fixed the conflicts and added the notes

**Pull request merged**